### ISSUE TITLE
0 errorCode returns when data part not exist in existing response

### ISSUE THREAD

**Issue by starspire** [EspeoBlockchain/gardener-server#18]
When data part doesn't exists in existing response oracle should return 404 not 0 errorCode.

Currently for:
```
//misspell, correctly should be .chartName
json(https://api.coindesk.com/v1/bpi/currentprice.json).chartname
```
is:
```
value: ""
error: 0
```
should be:
```
value: ""
error: 4004
```

**Comment by kss-espeo** [comment]
This is likely closely related to https://github.com/EspeoBlockchain/gardener-server/issues/24

**Comment by starspire** [comment]
Fixed in #26
